ai-ml/emr-spark-rapids/examples/xgboost/execute_spark_rapids_xgboost.sh

```diff
@@ -10,15 +10,18 @@
 read -p "Enter EMR Virtual Cluster AWS Region: " AWS_REGION
 read -p "Enter the EMR Virtual Cluster ID: " EMR_VIRTUAL_CLUSTER_ID
 read -p "Enter the EMR Execution Role ARN: " EMR_EXECUTION_ROLE_ARN
+read -p "Enter the customized Docker image URI: " XGBOOST_IMAGE
 read -p "Enter the CloudWatch Log Group name: " CLOUDWATCH_LOG_GROUP
-read -p "Enter the S3 Bucket for storing PySpark Scripts, Pod Templates and Input data. For e.g., s3://<bucket-name>: " S3_BUCKET
+read -p "Enter the S3 Bucket (Just the Bucket Name) for storing PySpark Scripts, Pod Templates, Input data and Output data. For e.g., <bucket-name>: " S3_BUCKET
 
 #--------------------------------------------
 # DEFAULT VARIABLES CAN BE MODIFIED
 #--------------------------------------------
 JOB_NAME='spark-rapids-emr'
 EMR_EKS_RELEASE_LABEL="emr-6.10.0-spark-rapids-latest"
-XGBOOST_IMAGE="public.ecr.aws/o7d8v7g9/emr-6.10.0-spark-rapids:0.11"
+#XGBOOST_IMAGE="public.ecr.aws/o7d8v7g9/emr-6.10.0-spark-rapids:0.11"
```

Comment:
Should this be commented on?

Reply:
Yes, it should be commented out or even deleted out as the Docker image should be supplied by the user. The public image I have is on my Isengard account that cannot be maintained permanently.